src/main/resources/org/openstreetmap/atlas/checks/validation/tag/invalid-tags-check-filter.json

```diff
@@ -0,0 +1,52 @@
+{
+  "filters": [
+    "island->*&natural->spring,hot_spring&name->*",
```

Comment:
If we go the resource route, the default values should be the tags in the original version

Reply:
@danielduhh https://github.com/osmlab/atlas-checks/pull/296/commits/96e197a3683865b16b6b502b8a558d0714fa312f should address this.